packages/app/src/routers/ViewRouter/ViewRouter.tsx

```diff
@@ -44,10 +42,6 @@ export interface ViewProps extends RouteComponentProps {
 }
 const View = (props: ViewProps) => {
   (window as any).ITWIN_VIEWER_HOME = window.location.origin;
-  const config = useConfig();
```

Comment:
you didnt want to kill the associated buddi region code in the config?

Reply:
No, we still use that all around (use URL prefix), wanted to go with minimal changes.